gridt/src/app/movements/add-movement/add-movement.page.spec.ts

```diff
@@ -48,12 +48,131 @@ describe('AddMovementPage', () => {
     fixture.detectChanges();
   }));
 
-  it('should create', () => {
+  it("should create", () => {
     expect(component).toBeTruthy();
   });
 
-  it('should dismiss the alert when leaving the page', () => {
+  it("should dismiss the alert when leaving the page", () => {
     component.ngOnDestroy();
     expect(alertSpy.dismiss).toHaveBeenCalled();
   });
+
+  it("should not disable the confirm button when the form is valid", () => {
+    const submitEl = fixture.debugElement.nativeElement.querySelector("#confirmCreation");
+    const compiled = fixture.componentInstance;
+    compiled.form.patchValue({
+      name: "test",
+      short_description: "this is a short description",
+      description: "this is a longer description that the short description to be able to tell the difference",
+      interval: "daily",
+      category: "Community",
+      location: "Groningen"
+    });
+    fixture.detectChanges();
+    expect(submitEl.disabled).not.toBeTruthy();
```

Comment:
`submitEl.disabled` should actually be false right? Not just Falsy, which is a broader statement I think.

Reply:
See also [this question on stackoverflow](https://stackoverflow.com/questions/32615713/tobetrue-vs-tobetruthy-vs-tobetrue)

Reply:
I will look into this

Reply:
That seems to be the correct way of doing this, so i will change it.